databricks/koalas/frame.py

```diff
@@ -4344,6 +4344,22 @@ def drop(self, labels=None, axis=1,
         0  1  7
         1  2  8
 
+        >>> pdf = pd.DataFrame({'x': [1, 2], 'y': [3, 4], 'z': [5, 6], 'w': [7, 8]},
+        ...                    columns=['x', 'y', 'z', 'w'])
+        >>> columns = [('a', 'x'), ('a', 'y'), ('b', 'z'), ('b', 'w')]
+        >>> pdf.columns = pd.MultiIndex.from_tuples(columns)
+        >>> kdf = ks.DataFrame(pdf)
+        >>> kdf  # doctest: +NORMALIZE_WHITESPACE
```

Comment:
Shall we skip pandas part?

```py
>>> df = ks.DataFrame({'x': [1, 2], 'y': [3, 4], 'z': [5, 6], 'w': [7, 8]},
...                   columns=['x', 'y', 'z', 'w'])
>>> columns = [('a', 'x'), ('a', 'y'), ('b', 'z'), ('b', 'w')]
>>> df.columns = pd.MultiIndex.from_tuples(columns)
>>> df  # doctest: +NORMALIZE_WHITESPACE
```

Reply:
@ueshin Sure, It looks better! Thanks for review :)